src/components/FullTextView/FullTextView.spec.tsx

```diff
@@ -32,6 +33,22 @@ describe('FullTextView', () => {
     expect(component.find('[data-hook="popover-content"]').at(0).text()).toBe('Delete this super awesome thing?');
   });
 
+  it('should dynamically load and display full content on hover and hide it on leave in tooltip using `shouldLoadAsync` prop', async () => {
+    const content = (
+      <div>
+        <span>Delete this super awesome thing</span>
+        <i>?</i>
+      </div>
+    );
+    const component = mount(<FullTextView shouldLoadAsync maxWidth={0}>{content}</FullTextView>);
+
+    expect(component.find('[data-hook="popover-content"]').length).toBe(0);
```

Comment:
why not using the driver instead?

Reply:
We had this approach before, so I just added 1 more test with the same shape. I didn't have an intention to refactor it in this PR.